cmd/add.go

```diff
@@ -64,12 +64,15 @@ func addHook(hookName string, fs afero.Fs) {
 if [ "$LEFTHOOK" = "0" ]; then
   exit 0
 fi
+
+exec < /dev/tty # <- enables interactive shell
```

Comment:
This is useful when you run interactive scripts by pre-commit hook https://unix.stackexchange.com/a/91716